CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,27 @@
+## The Arbor Code of Conduct
+
+* We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or any other characteristic.
+* Please avoid using overtly provoking aliases or nicknames that might detract from a friendly, safe and welcoming environment for all.
+* Please be kind and courteous. There’s no need to be mean or rude.
+* Respect that people have differences of opinion and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a right answer.
```

Comment:
Just in terms of content: I wouldn't say 'There is seldom a right answer.' It's true of course that there are almost always trade-offs, but I don't want the document to convey the impression that this means that most changes cannot be evaluated in an objective way, even in principle.

Reply:
"There may not be a right answer." or leave it out altogether?

Reply:
"There may not always be a right answer."

Reply:
I'd just leave it out.